docs/docs/en/guide/task/datax.md

```diff
@@ -19,6 +19,8 @@ DataX task type for executing DataX programs. For DataX nodes, the worker will e
 - **Environment Name**: Configure the environment name in which to run the script.
 - **Number of failed retry attempts**: The number of times the task failed to be resubmitted.
 - **Failed retry interval**: The time, in cents, interval for resubmitting the task after a failed task.
+- **Cpu quota**: Assign the specified CPU time quota to the task executed. Takes a percentage value. Default -1 means unlimited. For example, the full CPU load of one core is 100%,and that of 16 cores is 1600%. This function is controlled by [task.resource.limit.state](https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/architecture/configuration.html)
```

Comment:
we should avoid using `lastest` in our document, and should use place hold `<version>` then replace them in our release process. for more detail see https://dolphinscheduler.apache.org/en-us/community/release-prepare.html section "Update Version"
```suggestion
- **Cpu quota**: Assign the specified CPU time quota to the task executed. Takes a percentage value. Default -1 means unlimited. For example, the full CPU load of one core is 100%,and that of 16 cores is 1600%. This function is controlled by [task.resource.limit.state](https://dolphinscheduler.apache.org/en-us/docs/<version>/user_doc/architecture/configuration.html)
```

Reply:
change to ```<version>``` may cause dead link check failed....Is there a workaround?

Reply:
🤣

Reply:
It works. Thanks.